packages/cli/src/ErrorReporting.ts

```diff
@@ -69,7 +69,7 @@ export const initErrorHandling = async () => {
 
 		if (
 			originalException instanceof QueryFailedError &&
-			originalException.message.includes('SQLITE_FULL')
+			['SQLITE_FULL', 'SQLITE_IOERR'].some((errMsg) => originalException.message.includes(errMsg))
```

Comment:
Are these errors still logged to console for example? They are noise in Sentry, but for someone maintaining the n8n instance it's probably something you would be interested in knowing

Reply:
these were already not being logged. but now that they are not being sent to Sentry either, maybe we should start logging these? 🤔